### ISSUE TITLE
Download several images in parallel

### ISSUE THREAD

**Issue by pmartin** [pmartin/plop-reader#86]
For now, when syncing new entries from the server, each time a new entry is saved to the local database, its preview image is downloaded -- which means *saving entries* takes a long time, as images are downloaded one at a time.

An idea to speed this up (consequence: speed up the "synchronization" duration) would be to download several images in parallel (like already done for EPUB files), using a thread pool or something like that.
